sdks/python/pom.xml

```diff
@@ -39,131 +39,165 @@
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>mkdir</executable>
+                  <arguments>
+                    <argument>-p</argument>
+                    <argument>${python.user.base}</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>setup-test-tox</id>
+                <phase>process-test-resources</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>${python.pip.bin}</executable>
+                  <arguments>
+                    <argument>install</argument>
+                    <argument>--user</argument>
+                    <argument>--upgrade</argument>
+                    <argument>--ignore-installed</argument>
+                    <argument>tox</argument>
+                  </arguments>
+                  <environmentVariables>
+                    <PYTHONUSERBASE>${python.user.base}</PYTHONUSERBASE>
+                  </environmentVariables>
+                </configuration>
+              </execution>
+              <execution>
+                <id>setuptools-test</id>
+                <phase>test</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>${python.user.base}/bin/tox</executable>
+                  <arguments>
+                    <argument>-e</argument>
+                    <argument>py27</argument>
+                    <argument>-c</argument>
+                    <argument>tox.ini</argument>
+                  </arguments>
+                  <environmentVariables>
+                    <PYTHONUSERBASE>${python.user.base}</PYTHONUSERBASE>
+                  </environmentVariables>
+                </configuration>
+              </execution>
+              <execution>
+                <id>setuptools-sdist</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+                <configuration>
+                  <executable>${python.interpreter.bin}</executable>
+                  <arguments>
+                    <argument>setup.py</argument>
+                    <argument>sdist</argument>
+                    <argument>--dist-dir</argument>
+                    <argument>${project.build.directory}</argument>
+                  </arguments>
+                  <environmentVariables>
+                    <PYTHONUSERBASE>${python.user.base}</PYTHONUSERBASE>
+                  </environmentVariables>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>beam-sdks-python-pydoc</id>
+      <activation>
```

Comment:
should be auto-activated by the release plugin and/or part of the existing release profile